Makefile

```diff
@@ -9,6 +9,7 @@
 SIL ?= @
 MAKEFLAGS += --no-print-directory
 
+
```

Comment:
why? :P

Reply:
I moved `CFLAGS` below line `include ../phoenix-rtos-build/Makefile.common`
Currently, in Makefile.common `CFLAGS` are reset.

This new line is a residue :P